README.md

```diff
@@ -30,9 +30,8 @@
 ![Logo](https://www.google.com/s2/favicons?sz=16&domain=breezometer.com) [BreezoMeter](https://breezometer.com) |  | Haifa | [@ofeka_dev](https://twitter.com/@ofeka_dev)
 ![Logo](https://www.google.com/s2/favicons?sz=16&domain=https://www.careboxhealth.com) [Carebox](https://www.careboxhealth.com) |  |  Tel Aviv | [@jodoron](https://twitter.com/jodoron)
 ![Logo](https://www.google.com/s2/favicons?sz=16&domain=cnvrg.io) [cnvrg.io](https://cnvrg.io) |  | Jerusalem | [@jodoron](https://twitter.com/jodoron)
-![Logo](https://www.google.com/s2/favicons?sz=16&domain=www.withconfetti.com) [Confetti](https://www.withconfetti.com/) |  | Tel Aviv | [@jodoron](https://twitter.com/jodoron)
+![Logo](https://www.google.com/s2/favicons?sz=16&domain=https://www.withconfetti.com) [Confetti](https://www.withconfetti.com/) | ✓ | Tel Aviv | [@robozevel](https://twitter.com/robozevel)
 ![Logo](https://www.google.com/s2/favicons?sz=16&domain=constru.ai) [Constru.ai](https://constru.ai/) |  | Tel Aviv | [@silicakes](https://github.com/silicakes)
-![Logo](https://www.google.com/s2/favicons?sz=16&domain=www.coolcousin.com) [Cool Cousin](https://www.coolcousin.com) |  | Tel Aviv | [@jodoron](https://twitter.com/jodoron)
```

Comment:
@robozevel why was this entry for cool cousin removed?
- and another thing please ++ the total companies counter in the top part of the document